src/spyglass/spikesorting/v1/sorting.py

```diff
@@ -0,0 +1,318 @@
+from typing import Iterable
+
+import os
+import tempfile
+import time
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import spikeinterface as si
+import spikeinterface.extractors as se
+import spikeinterface.preprocessing as sip
+import spikeinterface.sorters as sis
+from spikeinterface.sortingcomponents.peak_detection import detect_peaks
+
+from spyglass.common.common_lab import LabMember, LabTeam
+from spyglass.common.common_interval import IntervalList
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.utils.dj_helper_fn import fetch_nwb
+from spyglass.spikesorting.v1.recording import (
+    SpikeSortingRecording,
+    SpikeSortingRecordingSelection,
+)
+from .recording import _consolidate_intervals
+
+schema = dj.schema("spikesorting_v1_sorting")
+
+
+@schema
+class SpikeSorterParameter(dj.Lookup):
```

Comment:
Similar to other conversation, do we want the table name and keys to stay plural for consistency?

Reply:
Yep will change these to plural as well

Reply:
done